docs/docs/reference/config-files/gatsby-config.md

```diff
@@ -206,6 +206,10 @@ module.exports = {
 
 You can specifiy the path of the generated TypeScript types file relative to the site root. Default: `src/gatsby-types.d.ts`.
 
+### gatsbyBuild
```

Comment:
Idea for a better name?

Reply:
Suggestions:

- `generateOnBuild?: boolean`

or more open (but too much YAGNI)

- `typegenPhases?: [string]` 

defaults to: `['develop']`
when i like tp add the `build` phase: `['develop','build']`

Reply:
I'd like to keep it a boolean flag, otherwise people can disable it through that for develop which I don’t want

Reply:
+1 for `generateOnBuild`, but I don't have a strong preference